README.md

```diff
@@ -80,15 +80,61 @@ manifests
     └── prometheus.package.yaml
 ```
 
-# Example
+# Building a catalog of Operators using `operator-registry`
 
-The [Dockerfile](upstream.Dockerfile) provides an example of using the `initializer` and `registry-server` to build a minimal container that provides an API over the example manifsts in [manifests](manifests).
+The [Dockerfile](upstream.Dockerfile) provides an example of using the `initializer` and `registry-server` to build a minimal container that provides a gRPC API over the example manifsts in [manifests](manifests).
 
 ```sh
-docker build -f upstream-example.Dockerfile -t example-registry:latest .
-docker run -p 50051:50051 example-registry:latest
+docker build -t example-registry:latest -f upstream-example.Dockerfile .
+docker push example-registry:latest
+```
+
+# Using the catalog with Operator Lifecycle Manager
+
+To add a catalog packaged with `operator-registry` to your cluster for use with [Operator Lifecycle Manager](https://github.com/operator-framework/operator-lifecycle-manager) (OLM) create a `CatalogSource` referencing the image you created and pushed aboved:
+
+```yaml
+apiVersion: operators.coreos.com/v1alpha1
+kind: CatalogSource
+metadata:
+  name: example-manifests
+spec:
+  sourceType: grpc
+  image: example-registry:latest
+```
+
+When you create the `CatalogSource`, watch the `catalog-operator` as part of your OLM deployment (assuming OLM is deloyed in a namespace called `olm`):
+
+```sh
```

Comment:
This highlights some pretty bad UX around catalogs... you should be able to watch a CatalogSource's status to know when it's ready.

What do you think about suggesting:

```sh
$ watch kubectl get packagemanifests
```
until we have improved the status of the CatalogSource?

(also just an implementation note - the package server and the catalog operator are separate binaries with separate connections to the registry pods. The catalog operator getting a connection doesn't necessarily mean the package server will also have a healthy connection, though they're usually correlated)